alot/settings/manager.py

```diff
@@ -91,8 +91,9 @@ def read_config(self):
         hooks_path = os.path.expanduser(self._config.get('hooksfile'))
         try:
             self.hooks = imp.load_source('hooks', hooks_path)
-        except:
+        except Exception as e:
             logging.debug('unable to load hooks file:%s', hooks_path)
+            logging.debug('because %s' % e)
```

Comment:
you could use [logging.exception](https://docs.python.org/2/library/logging.html#logging.Logger.exception) here..